projects/canopy/src/lib/header/header.component.ts

```diff
@@ -102,9 +102,15 @@ export class LgHeaderComponent implements AfterContentInit, OnDestroy {
     );
 
     if (this.accountMenuItems.length) {
+      // When the user shift + tabs of the first account menu item, which could be added asyncronously
       this.subscriptions.push(
-        // When a user shift + tabs on first account menu item on sm devices, focus the last logo not the primary nav
-        this.accountMenuItems.first.tabbedOut
+        this.accountMenuItems.changes
+          .pipe(startWith(this.accountMenuItems))
```

Comment:
Could we combine the three pipes?
```suggestion
          .pipe(
            startWith(this.accountMenuItems),
            switchMap((item: QueryList<LgAccountMenuListItemComponent>) => item.first.tabbedOut),
            filter((event: KeyboardEvent) => event.shiftKey && this.showResponsiveMenu),
          )
```

Reply:
Yeah, good point, will change that